partiql-conformance-tests/tests/fail_parse_select-from-where.rs

```diff
@@ -0,0 +1,7 @@
+use partiql_parser::lalr_parse;
+
+#[test]
+fn SFW_without_WHERE_expression() {
```

Comment:
Even if the generated tests don't get checked in, they will cause annoying warnings during compile. I recommend adding an `#[allow(non_snake_case)]` annotation.

Reply:
I think, perhaps, we shouldn't check them in.
Checking them in is duplicating the test data, potentially into a lot of code, that not all user might ever care to generate and/or run.

For that matter, perhaps this crate's `build.rs` should succeed even if the submodule hasn't been pulled down? (for the case that users just want to use the PartiQL code, not test it's conformance?)

Reply:
> I think, perhaps, we shouldn't check them in. Checking them in is duplicating the test data, potentially into a lot of code, that not all user might ever care to generate and/or run.
> 
> For that matter, perhaps this crate's `build.rs` should succeed even if the submodule hasn't been pulled down? (for the case that users just want to use the PartiQL code, not test it's conformance?)

The only issue I see with not having them checked-in is that we always have to deal with this implicit thing; on the flip side, having them checked-in, will give visibility on the effective tests currently on the fly. Agree with suppressing warnings and letting the build to proceed.

Reply:
> Regarding checking the tests in, I agree with the comments. Since there may be a lot of tests in the future, I'll not include the tests. If we find use in having the tests checked in and visible (e.g. for checking the GH actions logs), I can always check them in.

The question is: does the fact that we're going to have a lot of tests in the future is sufficient for us to skip test check-in?  Let's discuss about this further (non-blocking for this PR).